01_install_requirements.sh

```diff
@@ -24,65 +35,133 @@ fi
 
 # Install required packages
 # python-{requests,setuptools} required for tripleo-repos install
+if [ "${RHEL8}" = "y" ] ; then
+    sudo dnf install -y \
+        python36 \
+        python3-requests \
+        python3-setuptools
+    sudo alternatives --set python /usr/bin/python3
+
+    # TODO(russellb) - Install an rpm for this once OSP for RHEL8 is out
+    pushd ~
+    if [ ! -d crudini ] ; then
+        git clone https://github.com/pixelb/crudini
+    fi
+    pushd crudini
+    git pull -r
+    sudo pip3 install .
+    popd ; popd
+else
+    sudo yum -y install \
+        crudini \
+        python-pip \
+        python-requests \
+        python-setuptools
+fi
 sudo yum -y install \
-  crudini \
   curl \
   dnsmasq \
   golang \
   NetworkManager \
   nmap \
   patch \
   psmisc \
-  python-pip \
-  python-requests \
-  python-setuptools \
   vim-enhanced \
   wget
 
-# We're reusing some tripleo pieces for this setup so clone them here
-cd
-if [ ! -d tripleo-repos ]; then
-  git clone https://git.openstack.org/openstack/tripleo-repos
-fi
-pushd tripleo-repos
-sudo python setup.py install
-popd
+if [ "${RHEL8}" = "y" ] ; then
+    sudo subscription-manager repos --enable=ansible-2-for-rhel-8-x86_64-rpms
 
-# Needed to get a recent python-virtualbmc package
-sudo tripleo-repos current-tripleo
+    # make sure additional requirments are installed
+    sudo yum -y install \
+      ansible \
+      python3-netaddr \
+      redhat-lsb-core \
+      bind-utils \
+      jq \
+      libguestfs-tools \
+      libvirt \
+      libvirt-devel \
+      libvirt-daemon-kvm \
+      podman \
+      qemu-kvm \
+      virt-install \
+      unzip \
+      network-scripts \
+      ipmitool
 
-# There are some packages which are newer in the tripleo repos
-sudo yum -y update
+    sudo pip3 install yq
 
-# Setup yarn and nodejs repositories
-sudo curl -sL https://dl.yarnpkg.com/rpm/yarn.repo -o /etc/yum.repos.d/yarn.repo
-curl -sL https://rpm.nodesource.com/setup_10.x | sudo bash -
+    # TODO(russellb) - Install an rpm for this once OSP for RHEL8 is out
+    pushd ~
+    if [ ! -d virtualbmc ] ; then
+        git clone https://git.openstack.org/openstack/virtualbmc
+    fi
+    pushd virtualbmc
+    git pull -r
+    sudo pip3 install .
```

Comment:
This is certainly going to pull in a lot of packages.. If `--user` doesn't work for some reason, let's use a virtual environment.

Reply:
meh, I'm not that bothered by the global pip install stuff personally.  I'm fine if someone wants to clean this up as a follow-up.

We'll switch this to an rpm as soon as one is available.  We could probably just build one ourselves for this one.